service/controller/appcatalog/v1/resource/index/current.go

```diff
@@ -0,0 +1,33 @@
+package index
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/app-operator/service/controller/appcatalog/v1/key"
+)
+
+func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interface{}, error) {
+	cr, err := key.ToCustomResource(obj)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	name := configMapName(cr.Name)
+
```

Comment:
Missing a log line here
```suggestion
r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("finding index configMap %#q", name))
```

Reply:
good catch